R/resample-third-party-hazard-maps-to-zillow-grid.R

```diff
@@ -0,0 +1,22 @@
+# Resample the Fire risk map to be at the same resolution/extent as the Zillow data
+
+library(tidyverse)
+library(raster)
+library(rasterVis)
+
+template <- raster::raster("data/EmptyGrid/Empty_250_US.tif")
+
+# Source: https://www.fs.usda.gov/rds/archive/catalog/RDS-2016-0034
+fire <- raster::raster("data/bp_file_NationalFSim/bp_201608301.tif")
+fire_resamp <- raster::resample(x = fire, y = template, method = "bilinear")
+raster::writeRaster(x = fire_resamp, filename = "output/hazards-on-zillow-grid/fire-hazard_fsim-burn-probability_zillow-grid.tif")
+
+# Source: ?
```

Comment:
Same comment here - can we download the earthquake data programmatically?

Reply:
yes! I poached the code from Joe to run the python gdalwarp using the gdalUtils `R` package